plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterManagerImpl.java

```diff
@@ -1384,7 +1384,15 @@ private String[] getServiceUserKeys(KubernetesClusterVO kubernetesCluster) {
         if (owner == null || owner.getType() == Account.Type.PROJECT) {
             owner = CallContext.current().getCallingAccount();
         }
-        String username = owner.getAccountName() + "-" + KUBEADMIN_ACCOUNT_NAME;
```

Comment:
@parkinr 
can you explain more ?
- without this PR, what's the username ?
- with this PR, what's the username ?

Reply:
thanks @parkinr for the reply.
However, the security issue is  not resolved I think.

@DaanHoogland 
we may need to treat this as a serious security issue.